package.json

```diff
@@ -2,20 +2,22 @@
   "name": "mozilla-infobar",
   "version": "0.2.0",
   "description": "Mozilla InfoBar",
-  "private": true,
   "repository": {
     "type": "git",
     "url": "https://github.com/mozilla/mozilla-infobar"
   },
+  "bugs": {
+      "url": "https://github.com/mozilla/mozilla-infobar/issues/new"
+  },
   "scripts": {
-    "test": "karma start tests/unit/karma.conf.js"  
+    "test": "karma start tests/unit/karma.conf.js"
   },
   "author": "Mozilla",
   "license": "MPL2",
```

Comment:
Not related to this change, running `npm install` gives the following warning:

    npm WARN package.json mozilla-infobar@0.2.0 license should be a valid SPDX license expression

Should this value be `MPL-2.0`?